kernel/data-pipeline/cdc/protocol/src/main/proto/CDCResponseProtocol.proto

```diff
@@ -51,6 +51,22 @@ message CreateSubscriptionResult {
   bool existing = 2;
 }
 
+message NullValue {
+
+}
+
+message BigIntegerValue {
+  bytes value = 1;
+}
+
+message BigDecimalValue {
+  string value = 1;
+}
+
+message ClobValue {
+  string value = 1;
+}
+
 message FetchRecordResult {
```

Comment:
Looks `FETCH_RECORDS` was removed in request protocol, does `FetchRecordResult` need to be updated?